src/components/message/message-test.stories.tsx

```diff
@@ -55,3 +55,25 @@ Default.args = {
   children: "This is some information from the Message Component.",
   showCloseIcon: true,
 };
+
+export const Transparent = (args: MessageProps) => {
```

Comment:
**issue**: currently this test component is not showing up on Storybook as a test story, it needs to be added on line 8, like so `includeStories: ["Default", "Transparent"],` 👍